### PR TITLE
Use CSS helpers to toggle visiblity

### LINKED ISSUE #8767: use css helpers to toogle visiblity
maybe use css helpers to toogle visiblity?

_Originally posted by @daiagi in https://github.com/kodadot/nft-gallery/pull/8758#discussion_r1436785215_

replace `isMobile` and `!isMobile` with visibility helper when we can

- https://tailwindcss.com/docs/visibility#breakpoints-and-media-queries

better to use css than js

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Closes #8767
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by exezbcz** [comment]
why draft @pbkompasz ? anything still needed to be done?

**Comment by exezbcz** [comment]
can you please check? @kodadot/code-review-guild 

if the pr does not have the quality we can probably close

**Comment by yangwao** [comment]
@prury is everything working in this PR?

File: components/common/ShareDropdown.vue
Comment (prury) [inline components/common/ShareDropdown.vue:1]: button size on mobile: 
https://github.com/kodadot/nft-gallery/assets/36627808/9d6d4b1b-c4c2-4ce9-9ed9-c46c19731892

File: components/gallery/GalleryItem.vue
Comment (prury) [inline components/gallery/GalleryItem.vue:1]: gallery item: 
https://github.com/kodadot/nft-gallery/assets/36627808/1b5bc3b2-eb44-42c7-b64f-06f6a5f8f115

File: components/transfer/Transfer.vue
Comment (prury) [inline components/transfer/Transfer.vue:1]: desktop: 
https://github.com/kodadot/nft-gallery/assets/36627808/740cb924-e3fc-4b9e-9b21-be593a63c297

**Comment by prury** [comment]
> @prury is everything working in this PR?

nope

**Comment by yangwao** [comment]
Closing, anyone else can take it

**Pull request closed without merging**